### PR TITLE
Posts show date and Time of post

### PR DESCRIPTION
**Pull request by BertZZ**

### CODE REVIEW

File: spec/features/posts_have_dates_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```
[the remark this answers is not in the document]
Comment (neoeno) [reply spec/features/posts_have_dates_spec.rb:1]: Ignore this one if you like

**Comment by adamjohnsnow** [comment]
I approve this pull! Consider yourself reviewed.

**Comment by BertZZ** [comment]
Hang on the tests pass but it is still not working when I try and post a comment on the localhost version

**Comment by BertZZ** [comment]
I have changed the same Line in all 3 environments. It is definitely running in BST on the tests but not on the development environment. Maybe the localhost isn't running on the development environment??

**Comment by BertZZ** [comment]
Fixed It :)

**Comment by neoeno** [comment]
@BertZZ Code all looks reasonable.

Have a go at resolving the conflicts with the new master branch — then we'll be good to go.

**Comment by neoeno** [comment]
Thanks for the thorough review from @adamjohnsnow too!

**Comment by BertZZ** [comment]
@neoeno merging is still blocked. I think you may have to review it again

**Comment by neoeno** [comment]
Hound is out of control!

**Review by neoeno — APPROVED** [review]

**Pull request merged**